src/prisma/schema.prisma

```diff
@@ -2098,3 +2099,13 @@ model ReferralCodeUseEvent {
   @@index([referrerId])
   @@index([refereeId])
 }
+
+model ExternalProjects {
+  id        String  @id @default(uuid()) @db.Uuid
+  metadata  Json
+  recipient String
+  schemaId  String?
+  // Fields specific to ceramic
+  ceramicId String? @unique
+  source    String
```

Comment:
Suggestion; Add an EcosystemURL field, which is the @unique url for this project, and will also go into the final credential we create